### PR TITLE
Puts evidence secure lockers in evidence room in brig on Cerestation, delta, meta and on box station

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
 Pretty much what the title says

## Why It's Good For The Game
 Security can put the high-class contraband in the safe and no need to worry about antags to raid evidence with almost the whole armory of contraband, once its in the safe in mind that will be harder to break in to the secure evidence safe

## Images of changes

Screenshot 2024-02-13 123643
Screenshot 2024-02-13 123512
Screenshot 2024-02-13 123430
Screenshot 2024-02-13 123332

## Testing
 spawned as the warden and went to evidence to check if a secure lockers named evidence lockers was there on Boxstation, delta, meta and cere station

## Changelog
:cl:
tweak: Evidence lockers in brig are now locked and anchored with general brig access
/:cl:

### CODE REVIEW

**Comment by Coolrune206** [comment]
These safes are laughably ineffective due to being hackable very easily.
I'd personally favour combination safes like in the vault, with the codes being provided in the Wardens office. Can still crack it, steal the codes, or use a drill, but at least those take time, effort, and pose a risk.

**Comment by Probablyjay19** [comment]
You know these safes can be opened in seconds with a screwdriver and a multitool, correct?

**Comment by matttheficus** [comment]
https://hackmd.io/@matttheficus/HkpsVBvNs

One day 😔

**Comment by Fluffyzack21** [comment]
> These safes are laughably ineffective due to being hackable very easily. I'd personally favour combination safes like in the vault, with the codes being provided in the Wardens office. Can still crack it, steal the codes, or use a drill, but at least those take time,

> You know these safes can be opened in seconds with a screwdriver and a multitool, correct?

Yeah, I will fix it

**Comment by Fluffyzack21** [comment]
> You know these safes can be opened in seconds with a screwdriver and a multitool, correct?

fixed

**Comment by 1080pCat** [comment]
would the paper the warden gets show the codes to capts floor safe, and the vault safe?

**Comment by Fluffyzack21** [comment]
> would the paper the warden gets show the codes to capts floor safe, and the vault safe?

No, the paper would only show evidence safe

**Comment by Qwertytoforty** [comment]
I would prefer ID locked lockers, or back to the wall safes. I am not fine with full on safes that must be drilled or stethoscoped opened.

**Review by S34NW — APPROVED** [review]

File: code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm [hunk 1]
```diff
@@ -96,6 +96,12 @@
 	icon_state = "sec"
 	open_door_sprite = "sec_door"
 
+/obj/structure/closet/secure_closet/evidence
```
Comment (S34NW) [inline code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm:99]: Please make this like the cyberiad detective's locker:
Grey theme, anchored.
image

Please also make a special detective subtype with `ACCESS_FORENSICS_LOCKERS` and replace the var-edited detective closet with the new one you make.

Comment (Fluffyzack21) [reply code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm:99]: Screenshot 2024-02-13 115349
Like this?

Comment (Contrabang) [reply code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm:99]: there should be 2 types, that type but with the normal sec access, and a subtype with the forensics_locker access

Comment (S34NW) [reply code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm:99]: nearly there, let me put a suggestion for how to finish it

File: code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm [hunk 2]
```diff
@@ -96,6 +96,12 @@
...
+	name = "evidence locker"
+	req_access = list(ACCESS_SEC_DOORS)
+	icon_state = "sec"
+	open_door_sprite = "sec_door"
```
Comment (S34NW) [inline code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm:103]:
```suggestion
	anchored = TRUE
```

File: code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm (hunk 1 above)
Comment (S34NW) [reply code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm:99]: https://github.com/ParadiseSS13/Paradise/pull/24037/files#r1488356219

Comment (Contrabang) [reply code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm:99]: Could you revert the icon_state back to the grey state, and anchor it please? Also, these 2 additions should be near each other in the files, and not in the middle of a definition for another type (in this case, between /obj/structure/closet/secure_closet/security/ and /obj/structure/closet/secure_closet/security/populate_contents()).

I'd recommend putting both right after /obj/structure/closet/secure_closet/security/populate_contents()

Comment (Fluffyzack21) [reply code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm:99]: fixed the issue

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 3]
```diff
@@ -56195,6 +56189,9 @@
 	icon_state = "1-2"
 	},
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
+/obj/item/paper/safe_code{
```
Comment (S34NW) [inline _maps/map_files/MetaStation/MetaStation.dmm:56192]: You need to remove the warden safe code from meta

Comment (Fluffyzack21) [reply _maps/map_files/MetaStation/MetaStation.dmm]: fixed

**Review by Contrabang — COMMENTED** [review]
Code is Fine, CL is fine, and mapping looks fine from me, should be good to go soon

**Pull request merged**